modules/_labs/color-picker/react/stories/stories_ColorPicker.tsx

```diff
@@ -92,8 +92,7 @@ storiesOf('Labs|Color Picker/React', module)
     ];
 
     const onBlur = (e: React.FocusEvent<HTMLInputElement>) => {
-      // @ts-ignore
-      if (!popupRef.current || !popupRef.current.popper.popper.contains(e.relatedTarget)) {
+      if (!popupRef.current || !popupRef.current.contains(e.relatedTarget as Node)) {
```

Comment:
I'm unsure why `.contains` is so restrictive. It expects a `Node` which isn't directly compatible with `HTMLInputElement` without casting... Oh well, this removes the need for the `// @ts-ignore`

Reply:
This is still not an ideal implementation. If you focus on the input, it will open the color picker, but hitting tab goes into the ColorPicker and it will stay open until you focus on the Color Input again and focus on something _not_ inside the color picker. Not an intuitive way to close it, but I didn't want to mess with this too much.